pkg/kudoctl/cmd/install/install.go

```diff
@@ -36,6 +42,26 @@ func CmdErrorProcessor(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func validateInstallParameters() error {
+	if vars.Parameter != nil {
```

Comment:
As mentioned before, but: a Parameter struct would be really useful for encapuslating this information, with an IsValid method that returns (bool, error).

Reply:
^^